.github/workflows/test.yml

```diff
@@ -0,0 +1,84 @@
+name: Test
+
+on:
+  push:
+  pull_request:
+  schedule:
+  - cron: 0 4 * * *
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        python-version:
+        - '3.9'
+        - '3.10'
+        - '3.11'
+        - '3.12'
```

Comment:
Is it necessary to test against all of these older versions?

Reply:
Probably not, but I don’t know which version is powering our instance, so I just sticked to the versions tested in other repositories. See https://github.com/metabrainz/musicbrainz-weblate-checks/pull/2#issuecomment-1802241773